package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "new-lexeme-special-page",
+  "private": true,
+  "version": "0.0.1",
+  "description": "Form for the upcoming revival of the NewLexeme special page on Wikidata",
+  "main": "main.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wmde/new-lexeme-special-page.git"
+  },
+  "keywords": [
+    "Wikidata"
+  ],
+  "author": "The Wikidata team",
+  "license": "GPL-2.0-only",
```

Comment:
Should it really be GPL2 *only*? I thought the default is or-later.

Reply:
Mh, our internal guideline doesn't say. I'll look at what we're using in other places.

Reply:
We don't seem to have this field in most `package.json` files. But the first WikibaeLexeme php file that I opened had a license tag for `GPL-2.0-or-later`, so I'm using that here as well

Reply:
Done.